modules/monitoring-configuring-metrics-collection-profiles.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring-metrics-collection-profiles_{context}"]
+= Configuring metrics collection profiles
```

Comment:
**Suggestion**: Avoid using gerunds in concept titles. Refer to: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-concepts and https://redhat-documentation.github.io/modular-docs/#creating-concept-modules.

Reply:
That's a good point and good for you to raise it here, but because the feature is called "metrics collection profiles" I think it's best to use that phrase in the title, so I'm going to leave it as-is

Reply:
@bburt-rh I suggest the heading as: **Configuration of  metrics collection profiles**

That is more suitable here. I trust you to consider and implement it. WIth that, I approve the PR.